app/views/agent/sign_up_complete.scala.html

```diff
@@ -29,7 +29,8 @@
 
     @main_template(title = Messages("agent.sign-up-complete.title")) {
 
-        @dataHelper("income-source", summary.incomeSource.map(_.source))
+        @dataHelper("self-employment", summary.incomeSource.map(_.selfEmployment.toString))BusinessAccountingMethodControllerSpec
```

Comment:
```suggestion
        @dataHelper("self-employment", summary.incomeSource.map(_.selfEmployment.toString))
```
I don't think that is supposed to be there :)

Reply:
fixed